tools/kvrocks2redis/parser.cc

```diff
@@ -323,9 +331,12 @@ rocksdb::Status WriteBatchExtractor::DeleteCF(uint32_t column_family_id, const S
         break;
       }
       case kRedisSortedint: {
-        if (!to_redis_) {
-          command_args = {"SIREM", user_key, std::to_string(DecodeFixed64(sub_key.data()))};
+        std::string sub_key_str=std::to_string(DecodeFixed64(sub_key.data()));
+        std::string cmd_str="SIREM";
+        if (to_redis_) {
+          cmd_str="ZREM";
```

Comment:
Using `? :` may be more concise.
```suggestion
        cmd_str = to_redis_ ? "SREM" : "SIREM";
```